src/impl/peerconnection.cpp

```diff
@@ -994,7 +1004,8 @@ void PeerConnection::processLocalDescription(Description description) {
 	}
 
 	// Set local fingerprint (wait for certificate if necessary)
-	description.setFingerprint(mCertificate.get()->fingerprint());
+	description.setFingerprint(mCertificate.get()->fingerprint(),
+	                           CertificateFingerprint::Algorithm::Sha256);
```

Comment:
`Certificate::fingerprint()` should return a `CertificateFingerprint` with `Sha256` as the caller should not have to infer the algorithm.

Reply:
👍